static/src/javascripts/boot.js

```diff
@@ -72,14 +72,28 @@ const go = () => {
             });
         }
 
+        const useStandaloneBundle = window.guardian.config.tests.standaloneCommercialBundleVariant === "variant";
```

Comment:
i think this should use [`lib/config`](https://github.com/guardian/frontend/blob/main/static/src/javascripts/lib/config.js)?

Reply:
If the config is strongly typed, do we need to? Dotcom-rendering does it this way.

Happy to change it whilst `boot.js` isn’t converted (will it ever?)